mitiq/typing.py

```diff
@@ -16,21 +16,30 @@
 
 from collections import Counter
 from dataclasses import dataclass
-from typing import Any, Dict, List, Optional, Sequence, Tuple, Union
+from enum import Enum
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Tuple, Union
 
 import numpy as np
 import numpy.typing as npt
 from cirq import Circuit as _Circuit
 
+
+class EnhancedEnum(Enum):
+    # This is for backwards compatibility with the old representation
+    # of SUPPORTED_PROGRAM_TYPES, which was a dictionary
+    @classmethod
+    def keys(cls) -> Iterable[str]:
+        return [member.value for member in cls]
```

Comment:
Good idea with this to ensure the PR is small and understandable! Am I right in that this will be removed in a follow on PR that will be included in the next release as well?